packages/jetpack-blocks/src/editor-shared/get-site-fragment.js

```diff
@@ -19,13 +9,6 @@ export default function getSiteFragment() {
 		return window._currentSiteId;
 	}
 
-	// Calypso will contain a site slug or ID in the site fragment.
-	// WP.org will contain either `post` or `post-new.php`.
-	const siteFragment = getCalypsoSiteFragment( window.location.pathname );
-	if ( ! includes( [ 'post.php', 'post-new.php' ], siteFragment ) ) {
-		return siteFragment || null;
-	}
-
```

Comment:
It's all Calypso-specific. We're moving out of —and no longer building specifically for— Calypso.

Makes https://github.com/Automattic/wp-calypso/issues/31209 irrelevant

Reply:
If we restore this, we could land this while Gutenberg remains in Calypso by replacing the Calypso imports with `@automattic/jetpack-blocks/…` 🤔